pages/account/manage/milestone/[[...data]].js

```diff
@@ -53,15 +53,23 @@ export default function ManageMilestone({ BASE_URL, milestone }) {
     additionalMessage: "",
   });
   const [title, setTitle] = useState(
-    milestone.title || "Title of your Milestone"
+    milestone.title || ""
   );
   const [description, setDescription] = useState(
-    milestone.description || "Description of your Milestone"
+    milestone.description || ""
   );
-  const [url, setUrl] = useState(milestone.url || "");
-  const [icon, setIcon] = useState(milestone.icon || "FaGithub");
-  const [date, setDate] = useState(milestone.date || "");
-  const [isGoal, setIsGoal] = useState(milestone.isGoal ? true : false);
+  const [url, setUrl] = useState(
+    milestone.url || ""
+    );
+  const [icon, setIcon] = useState(
+    milestone.icon || ""
+    );
+  const [date, setDate] = useState(
+    milestone.date || ""
+    );
+  const [isGoal, setIsGoal] = useState(
+    milestone.isGoal ? true : false
+    );
```

Comment:
I think the code formatting style is not correct? Please check your Prettier plugin